### PR TITLE
CM-28234 - Add SBOM report to the readme

### PR DESCRIPTION
**Pull request by jenia-sakirko**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -711,6 +718,44 @@ In the example below, the command to ignore a specific SCA package is as follows
...
+Using this command you can create an SBOM report for your local project or for your repository URI.
+
+The following options are available for use with this command:
+| Option              | Description | Required | Default |
+|---------------------|-------------|----------|---------|
+| `-f, --format [spdx-2.2\|spdx-2.3\|cyclonedx-1.4]` | SBOM format | Yes | | 
+| `-o, --output-format [JSON]` | Specify the output file format | No | json |
+| `--output-file PATH` | Output file | No | autogenerated filename saved to the current directory |
+| `--include-vulnerabilities` | Include vulnerabilities | No | False |
+| `-include-dev-dependencies` | Include dev dependencies | No | False |
```
Comment (MarshalX) [inline README.md:735]: Ig - was missed. Should be double one

File: README.md [hunk 2]
```diff
@@ -711,6 +718,44 @@ In the example below, the command to ignore a specific SCA package is as follows
...
+For example:\
+`cycode report sbom --format spdx-2.3 --include-vulnerabilities --include-dev-dependencies repository_url https://github.com/cycodehq-public/cycode-cli.git`
+
+### Local Project
+
+To create an SBOM report for a path:\
+`cycode report sbom --format <sbom format> --include-vulnerabilities --include-dev-dependencies --output-file </path/to/file> path </path/to/project>`
+
+For example:\
+`cycode report sbom --format spdx-2.3 --include-vulnerabilities --include-dev-dependencies path ~/path/to/local/project`
```
Comment (MarshalX) [inline README.md:757]: Does ~ work for you btw? I think for me it can't be resolved. I'm using zsh

**Review by MarshalX — COMMENTED** [review]
Cool!

File: README.md (hunk 1 above)
Comment (jenia-sakirko) [reply README.md]: fixed

**Review by MaorDavidzon — APPROVED** [review]

**Pull request merged**